### PR TITLE
[IMP] mail: restore skipped moderation tests

### PR DESCRIPTION
**Pull request by seb-odoo**
task-2199306

### CODE REVIEW

File: addons/mail/static/tests/owl/component/discuss_tests.js
Comment (alexkuhn) [inline addons/mail/static/tests/owl/component/discuss_tests.js]: keep hard-coded value instead of storing in a variable
_(same comment in other unskipped tests)_

Comment (seb-odoo) [reply addons/mail/static/tests/owl/component/discuss_tests.js]: done

File: addons/mail/static/src/owl/components/moderation_reject_dialog/moderation_reject_dialog.xml
Comment (alexkuhn) [inline addons/mail/static/src/owl/components/moderation_reject_dialog/moderation_reject_dialog.xml]: why `id` instead of `class`?

Comment (seb-odoo) [reply addons/mail/static/src/owl/components/moderation_reject_dialog/moderation_reject_dialog.xml]: Habit for form elements to be able to use "label for".

Comment (alexkuhn) [reply addons/mail/static/src/owl/components/moderation_reject_dialog/moderation_reject_dialog.xml]: I would still not use `id` and prefer `class`

Comment (seb-odoo) [reply addons/mail/static/src/owl/components/moderation_reject_dialog/moderation_reject_dialog.xml]: If it's the only change you can do it and merge I suppose.

**Review by alexkuhn — APPROVED** [review]

**Comment by alexkuhn** [comment]
should be green, but waiting for green before merge just to be sure

**Pull request merged**
